adafruit_hts221.py

```diff
@@ -105,10 +112,10 @@ class Rate(CV):
 
 Rate.add_values(
     (
-        ("ONE_SHOT", 0, 0, None),
-        ("RATE_1_HZ", 1, 1, None),
-        ("RATE_7_HZ", 2, 7, None),
-        ("RATE_12_5_HZ", 3, 12.5, None),
```

Comment:
Did these `None`s serve any purpose previously?

Reply:
My understanding from other libraries that do this is the store the resolution or LSB.  As an example, if an option changes the sensitivity of a sensor to the equivalent of 8 bits, this would be 0.0039.  So it's just that it doesn't here because it changes rate, not anything where the value of a "unit" of it changes, if that makes sense.

Reply:
Did you run it with a device? Seems okay to me. But want to try to make sure it won't cause IndexError anywhere ifi we can. I don't have one of these sensors.

Reply:
I don't think I did, I don't think I own this sensor.  The other libraries use it in calculations (calculating temperature using the LSB value and digital value, like from ADC) but it never seemed to be used in communication with the sensor.  Happy to find a sensor and test with it if you'd like!

Reply:
Turns out the HTS221 is out of stock on Adafruit and Digi-Key with no restock date :/ depreciated maybe?